client/my-sites/plans/jetpack-plans/product-store/context/store-item-context.ts

```diff
@@ -0,0 +1,9 @@
+import { createContext, useContext } from 'react';
+import { StoreItemInfo } from '../types';
+
+const StoreItemInfoContext = createContext< StoreItemInfo | null >( null );
+
+export default StoreItemInfoContext;
+
+// A little hack here as we populate the default value in the component it is used.
+export const useStoreItemInfoContext = () => useContext( StoreItemInfoContext ) as StoreItemInfo;
```

Comment:
I would suggest to do this instead

```suggestion
const StoreItemInfoContext = createContext( {} as StoreItemInfo );

export default StoreItemInfoContext;

export const useStoreItemInfoContext = () => useContext( StoreItemInfoContext );
```